config.yaml

```diff
@@ -59,7 +59,8 @@ metamorph:
   statsKeypress: false # enable stats keypress. If enabled pressing any key will print stats to stdout
   profilerAddr: localhost:9992 # address to start profiler server on
   blocktxTimeout: 1s # timeout for blocktx service
-  healthServerDialAddr: localhost:8005
+  metamorphHealthServerDialAddr: localhost:8005
+  blocktxHealthServerDialAddr: localhost:8006
```

Comment:
I think this setting `blocktxHealthServerDialAddr` belongs to `blocktx` so it should be moved e.g. line 87 without the prefix `blocktx`